### PR TITLE
Update tutorial.md add funding

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: topics/ecology/tutorials/panoply_ebv/tutorial.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ contributors:
 - colineroyaux
 - Marie59
 - annefou
-
+funding:
```
Comment (hexylena) [inline topics/ecology/tutorials/panoply_ebv/tutorial.md:27]: you'll need to make the key about `contributions:` and add an `authorship:` for the four of you

File: topics/ecology/tutorials/gbif_cleaning/tutorial.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ tags:
 contributors:
 - yvanlebras
 - sbenateau
```
Comment (hexylena) [inline topics/ecology/tutorials/gbif_cleaning/tutorial.md:24]:
```suggestion
contributions:
  authorship:
  - yvanlebras
  - sbenateau
```

File: topics/ecology/tutorials/gbif_cleaning/tutorial.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ tags:
...
+funding:
+- pndb
```
Comment (hexylena) [inline topics/ecology/tutorials/gbif_cleaning/tutorial.md:26]:
```suggestion
contributions:
  authorship:
  - yvanlebras
  - sbenateau
  funding:
  - pndb
```

File: topics/ecology/tutorials/champs-blocs/tutorial.md [hunk 4]
```diff
@@ -19,6 +19,8 @@ contributors:
 - yvanlebras
 translations:
    - fr
+funding:
+- pndb
```
Comment (hexylena) [inline topics/ecology/tutorials/champs-blocs/tutorial.md:23]: must be a child of `contributions`

File: topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md [hunk 5]
```diff
@@ -21,6 +21,10 @@ contributors:
 - colineroyaux
 - Marie59
 - yvanlebras
+funding:
```
Comment (hexylena) [inline topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md:24]: same issue

File: topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md [hunk 6]
```diff
@@ -32,6 +32,9 @@ contributors:
 - colineroyaux
 - yvanlebras
 - jbmihoub
+funding:
```
Comment (hexylena) [inline topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md:35]: also here

File: topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md [hunk 7]
```diff
@@ -24,6 +24,9 @@ contributors:
 - ecoissac
 - fboyer
 - yvanlebras
+  funding:
```
Comment (hexylena) [inline topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md:27]: same

Comment (yvanlebras) [reply topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md:28]: oh yes, sorry... THANK YOU!

File: topics/ecology/tutorials/panoply_ebv/tutorial.md (hunk 1 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/panoply_ebv/tutorial.md]: thanks!

File: topics/ecology/tutorials/champs-blocs/tutorial.md (hunk 4 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/champs-blocs/tutorial.md]: thanks!

File: topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md (hunk 5 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md]: thank you very much!

File: topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md (hunk 6 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md]: sorry ;)

File: topics/ecology/tutorials/obisindicators/tutorial.md
[the remark this answers is not in the document]
Comment (yvanlebras) [reply topics/ecology/tutorials/obisindicators/tutorial.md:1]: Done!

**Review by hexylena — APPROVED** [review]
Perfect !

**Pull request merged**